libretro-buildbot-recipe.sh

```diff
@@ -238,6 +238,20 @@ buildbot_log() {
 	fi
 }
 
+buildbot_tag() {
+	# If the repository has no tags, it returns an empty string
+	TAG=$(git ls-remote --quiet --tags --sort="v:refname" | tail -n1 | sed 's/.*\///; s/\^{}//')
+	# Get the current latest tag
+	OLD_TAG=$(git describe --abbrev=0 --tags)
```

Comment:
You're going to want to add `--always` to `OLD_TAG`. If a repo has no tags the command will fail and print the following.
```
$ OLD_TAG=$(git describe --abbrev=0 --tags)
fatal: No names found, cannot describe anything.
$ echo $?
128
```
```
$ OLD_TAG=$(git describe --abbrev=0 --tags --always)
$ echo $?
0
```

Reply:
Done!